### PR TITLE
Docs changes to PR #4092

### PR DESCRIPTION
**Pull request by chenopis**
Follow-up editorial changes to PR #4092.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4224)

### CODE REVIEW

File: docs/admin/extensible-admission-controllers.md [hunk 1]
```diff
@@ -31,34 +30,34 @@ This page describes how to use Initializers and External Admission Webhooks.
...
 ### What are initializers?
 
-Two meanings:
+*Initializer* has two meanings:
 
 * A list of pending pre-initialization tasks, stored in every object's metadata
   (e.g., "AddMyCorporatePolicySidecar").
 
-* The controllers which actually perform those tasks. The name of the task
+* A controller, which actually perform those tasks. The name of the task
```
Comment (caesarxuchao) [inline docs/admin/extensible-admission-controllers.md]: Users are supposed to write these controllers. Is this message clear? Or shall we s/A controller/a user costomized controller/?

File: docs/admin/extensible-admission-controllers.md [hunk 2]
```diff
@@ -195,7 +193,7 @@ To turn it on, you need to
 
 ### Write a webhook admission controller
 
-See [here](https://github.com/caesarxuchao/example-webhook-admission-controller)
+See [caesarxuchao/example-webhook-admission-controller](https://github.com/caesarxuchao/example-webhook-admission-controller)
```
Comment (caesarxuchao) [inline docs/admin/extensible-admission-controllers.md]: @lavalamp you might want to take a look at my example. I'll consolidate it a little bit more tonight.

File: docs/admin/extensible-admission-controllers.md (hunk 1 above)
Comment (chenopis) [inline docs/admin/extensible-admission-controllers.md]: Good point, I'll make the change.

**Comment by chenopis** [comment]
I'm going to merge this.

**Pull request merged**